snmp_exporter/collector.py

```diff
@@ -18,7 +18,10 @@ def walk_oid(session, oid):
     while True:
       # getbulk starts from the last oid we saw.
       vl = netsnmp.VarList(netsnmp.Varbind('.' + last_oid))
-      if not session.getbulk(0, 25, vl):
+      result = session.getbulk(0, 25, vl)
+      # result check: when querying an OID that the target
+      # device does not support, getbulk returns ('',)
```

Comment:
Your PR description says endOfMibView, this says a MIB that's not supported. Which is it?